source/upgrade.txt

```diff
@@ -72,6 +72,23 @@ Version 3.0 Potential Breaking Change
 - The LINQ2 provider has been removed from this version of the driver.
   You must use LINQ3 for all LINQ queries.
 
+- Previous versions of the {+driver-short+} supported two GUID representation modes.
+  In version 3.0, ``GuidRepresentationMode.V3`` is the only supported mode. This change
+  has the following effects on the driver:
+  
+  - The methods in the ``BsonBinaryReader``, ``BsonBinaryWriter``, ``JsonReader``, and
+    ``JsonWriter`` classes no longer check the ``GuidRepresentationMode``.
+  - The ``BsonBinaryData(Guid)`` constructor has been removed. To construct a ``BsonBinaryData``
+    object from a GUID, use the ``BsonBinaryData.Create(Guid, GuidRepresentation)`` constructor.
+  - The ``BsonBinaryData.GuidRepresentation`` property has been removed. 
+  - You can call the ``BsonBinaryData.ToGuid()`` method only on ``BsonBinaryData``
+    objects of subtype 4. If the object has any other subtype, you must call the
+    ``BsonBinaryData.ToGuid(GuidRepresentation)`` method and specify the subtype. 
+
+  The preceding changes affect your application only if you serialize and deserialize
+  BSON documents directly. If you map your MongoDB documents only to :ref:`csharp-poco`,
+  the ``GuidRepresentationMode`` doesn't affect your application.
```

Comment:
S: somewhere in this entry link to the GUIDs page

Reply:
good call. meant to do that.